wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -83,19 +83,16 @@ private void dismissWelcomeView() throws Exception {
     }
 
     private void openFile(final String filename) throws Exception {
-
-        String testTag = "openfile";
```

Comment:
You need to leave the testTag.  We parse the log files on this to form the groups, then the string below is used as the result name.